Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
+    :param level: log level of the message
+    :type level: ``pyDKB.common.types.logLevel`` member
+    :param *args: additional prefixes (will be output between log
+                  level prefix and message body)
+    :type *args: str
```

Comment:
*args can be anything because they are converted to str. Question is, how to indicate this in `:type:`?
https://github.com/PanDAWMS/dkb/pull/188#discussion_r241459053 - some discussion on this point.

Reply:
Concerning the comment you mention -- why not use `object`? Right now I can not invent an examle of non-object entity.. doesn\`t mean it is not possible, though.

Here, it was specified as 'string' to state that it will be *treated* as string. So if one wants to be sure about what shall appear in the output -- it is better to use string values.